src/parser/content-parser.php

```diff
@@ -10,13 +15,39 @@
 use WP_Block_Type_Registry;
 use Symfony\Component\DomCrawler\Crawler;
 
+/**
+ * The content parser that would be used to transform a post into an array of blocks, along with their attributes.
```

Comment:
I'd remove the tense here:

```suggestion
 * Content parser used to transform a post into an array of blocks and associated attributes.
```